app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,22 +1,48 @@
 import React from "react";
 
 import { Formik, Form } from "formik";
-import { Button } from "neetoui";
+// import { Button } from "neetoui";
 import { Input, Textarea } from "neetoui/formik";
+import { Dropdown, Label, Toastr, Button } from "neetoui/v2";
 import * as yup from "yup";
 
-import notesApi from "apis/notes";
+import { CONTACTS } from "constants/notes";
 
-export default function NewNoteForm({ onClose, refetch }) {
-  const handleSubmit = async values => {
-    try {
-      await notesApi.create(values);
-      refetch();
-      onClose();
-    } catch (err) {
-      logger.error(err);
-    }
+export default function NewNoteForm({ onClose }) {
+  const handleSubmit = () => {
+    onClose();
+    Toastr.success("Note Added successfully!");
   };
+
+  const Dropdowns = () => (
+    <>
+      <Label className="mb-1 neeto-ui-text-black">Contacts</Label>
+      <div className="contacts-dropdown">
+        <Dropdown
+          buttonStyle="text"
+          label="Select Role"
+          position="bottom-end"
+          className="m-1"
+          name="contacts"
+        >
+          <li>{CONTACTS[0]}</li>
+        </Dropdown>
+      </div>
+
+      <Label className="mb-1 mt-4 neeto-ui-text-black">Tags</Label>
+      <div className="contacts-dropdown">
+        <Dropdown
+          buttonStyle="text"
+          label="Select Role"
+          position="bottom-end"
+          className="m-1"
+          name="tags"
+        >
+          <li>{CONTACTS[0]}</li>
+        </Dropdown>
+      </div>
+    </>
+  );
```

Comment:
Why do we need this splitting here? 
Also, please use `Select` component from `v2` here.

Reply:
Select component allows support for labels inbuilt.